nixos/modules/services/hardware/pid-fan-controller.nix

```diff
@@ -0,0 +1,185 @@
+{
+  lib,
+  config,
+  pkgs,
+  ...
+}:
+let
+  cfg = config.services.pid-fan-controller;
+  #floating point option
+  fOpt = (
```

Comment:
Just naming it `mkFloatingPointOption` or so would be preferable to such a short name IMO.

In this case specifically, I think we could do without this helper though. Its called just 3 times below